PMMain/Midas/MidasGUI/MidasCommon.h

```diff
@@ -173,6 +173,11 @@ struct point {
     point(int xVal = 0, int yVal = 0) : x(xVal), y(yVal) { }
 };
 
+struct ori_data {
```

Comment:
I don't particularly like this name. I assume ori is supposed to stand for orientation, but the only reason I can figure that out is because I see that roll, pitch and yaw are members of the struct. The name should be more standalone.